examples/widgets_minimal/src/main.rs

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! This example provides demostrations of building a Qt Widgets CXX-Qt application
+
+use cxx_qt_lib::QString;
+use cxx_qt_lib_extras::{QApplication, QPushButton};
+
+fn main() {
```

Comment:
we should have some kind of test as well, even if C++ to see if this works, unsure how so far....